apps/wing-console/console/app/src/analytics.ts

```diff
@@ -10,10 +10,18 @@ export interface Analytics {
 }
 
 export const createAnalytics = (options: CreateAnalyticsOptions): Analytics => {
-  const segment = new Segment(options.segmentWriteKey);
+  let segment: Segment;
   const sessionId = Date.now();
+  try {
+    segment = new Segment(options.segmentWriteKey);
+  } catch (error) {
+    console.debug("failed to initialize analytics", error);
```

Comment:
This should probably fail silently, or check if a DEBUG flag is enabled first before outputting anything.